cmd/deploy.go

```diff
@@ -437,12 +465,27 @@ func (c deployConfig) Configure(f fn.Function) (fn.Function, error) {
 	// Envs
 	// Preprocesses any Envs provided (which may include removals) into a final
 	// set
+	// TODO: remove pointers and use applyVector(f.Run.Envs, c.Env)
 	f.Run.Envs, err = applyEnvs(f.Run.Envs, c.Env)
 	if err != nil {
 		return f, err
 	}
 
-	// .Revision
+	// Enabled Integrations
```

Comment:
I'd vote for keeping the `-` notation, it is being used with kn cli as well.

Reply:
It looks like the `-` notation is still used internally. Personally, I don't mind the syntactic sugar of `enable` / `disable`. It is generic, yes, but that may also be a "good thing".